hail/src/main/scala/is/hail/methods/LocalLDPrune.scala

```diff
@@ -88,15 +95,52 @@ private case class BitPackedVectorLoader(fullRowPType: PStruct, callField: Strin
     val allHomVar = gtSum == 2 * nPresent
 
     if (allHomRef || allHet || allHomVar || nMissing == nSamples) {
-      None
+      null
     } else {
       val gtMean = gtSum.toDouble / nPresent
       val gtSumAll = gtSum + nMissing * gtMean
       val gtSumSqAll = gtSumSq + nMissing * gtMean * gtMean
       val gtCenteredLengthRec = 1d / math.sqrt(gtSumSqAll - (gtSumAll * gtSumAll / nSamples))
 
-      Some(BitPackedVector(locus, alleles, packs.result(), nSamples, gtMean, gtCenteredLengthRec))
+      BitPackedVector(locus, alleles, packs.result(), nSamples, gtMean, gtCenteredLengthRec)
+    }
+  }
+}
+
+private case class BitPackedVectorLoader(fullRowPType: PStruct, callField: String, nSamples: Int) {
```

Comment:
This is only used in the unstaged implementation, right?

Reply:
correct